board/drivers/interrupts.h

```diff
@@ -14,7 +14,7 @@ void unused_interrupt_handler(void) {
 
 #define NUM_INTERRUPTS 102U                // There are 102 external interrupt sources (see stm32f413.h)
 interrupt interrupts[NUM_INTERRUPTS];
-
+// cppcheck-suppress misra-c2012-20.7
```

Comment:
let's fix instead of suppress all of the 20.7s

Reply:
They are suppressed till next PR as this one is pretty big already.

Reply:
the diff from 20.7 should be pretty small, let's do it in this PR